adapters/ikea/tradfri_switch_on_off.py

```diff
@@ -18,6 +18,16 @@ def __init__(self, devices):
 
         self.devices.append(self.switch)
 
+    def handleMqttMessage(self, device_data, message):
+        if 'click' not in message.raw:
```

Comment:
This probably will filter all heartbeat messages if there are any

Reply:
Is there heartbeat messages on E1743 ?  I thought that all messages contain "click"...

Reply:
Have no idea as doesn't have this device, but usually zigbee devices which work on battery have such messages